package.json

```diff
@@ -37,33 +42,35 @@
     ]
   },
   "devDependencies": {
-    "babel-cli": "6.x",
-    "babel-core": "6.x",
-    "babel-preset-shopify": "15.0.1",
-    "eslint": "^3.19.0",
-    "eslint-index": "^1.3.0",
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.25.0",
+    "babel-preset-shopify": "^16.1.0",
+    "eslint": "^4.3.0",
+    "eslint-index": "^1.4.0",
     "eslint-plugin-shopify": "file:.",
-    "isparta": "4.x",
-    "istanbul": "0.4.4",
-    "mocha": "3.x"
+    "isparta": "^4.0.0",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.4.2",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
   },
   "peerDependencies": {
-    "eslint": "<4 >=3.17"
+    "eslint": "<5 >=4.3.0"
   },
   "dependencies": {
     "babel-eslint": "^7.2.3",
-    "eslint-plugin-ava": "^4.2.0",
-    "eslint-plugin-babel": "^4.1.1",
-    "eslint-plugin-chai-expect": "1.1.x",
-    "eslint-plugin-flowtype": "^2.32.1",
-    "eslint-plugin-import": "2.2.x",
-    "eslint-plugin-jsx-a11y": "^5.0.1",
-    "eslint-plugin-lodash": "^2.4.2",
-    "eslint-plugin-mocha": "^4.9.0",
-    "eslint-plugin-node": "^4.2.2",
+    "eslint-plugin-ava": "^4.2.1",
+    "eslint-plugin-babel": "^4.1.2",
+    "eslint-plugin-chai-expect": "^1.1.1",
+    "eslint-plugin-flowtype": "^2.35.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-jsx-a11y": "^6.0.2",
```

Comment:
`eslint-plugin-jsx-a11y`'s [`5.1.0`](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/CHANGELOG.md#510--2017-06-26) release added a few new rules. I think we should include and disabled them at the very least.